langlib/lang.function/src/main/ballerina/function.bal

```diff
@@ -21,6 +21,14 @@ import ballerina/jballerina.java;
 # If the arguments specified in `args` are not of the type required by `func`,
 # then this will panic.
 #
+# ```
+# function:call(getGreeting) ⇒ Hello
+#
+# function:call(getGreeting, "David") ⇒ Hello David!
+#
+# function getGreeting(string? name = ()) returns string => name is () ? "Hello" : string `Hello ${name}!`;
```

Comment:
We need to define this function at the top if we are running this examples on shell. WDYT?

Reply:
I am also think that, defining in the top will be better in this situation.

Reply:
Given that the user will often be looking for a way to use this with a function they've written I still think it's OK to have it at the end. However, I've moved this to the top since we added a panic example also.